src/styles.css

```
@@ -1,710 +1,270 @@
-═══════════════════════════════════════════ */
-@media(max-width:380px){
-  .nav-inner{padding:0 8px}
-  .compare-btn,.analytics-btn{display:none}
-  .cd-num{font-size:18px}
-  .stat-num{font-size:15px}
-  .stat-lbl{font-size:8px}
-  .org-card{padding:12px}
-  .org-logo{width:34px;height:34px}
-  .org-name{font-size:12px}
-}
+  animation:fadeUp .25s ease forwards;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)